src/main/resources/spring/camel-context.xml

```diff
@@ -213,12 +250,7 @@
                             </setHeader>
                         </when>
                         <otherwise>
-                            <setHeader headerName="Exchange.HTTP_RESPONSE_CODE">
-                                <simple resultType="String">${type:javax.servlet.http.HttpServletResponse.SC_NOT_FOUND}</simple>
-                            </setHeader>
-                            <setBody>
-                                <constant>Analysis not found</constant>
-                            </setBody>
+                            <throwException exceptionType="java.util.NoSuchElementException" message="Analysis not found"/>
```

Comment:
Our endpoints are using the term Report like in `/report/{id}` I think this message should be `Report not found` wdyt?

Reply:
+1

Reply:
+1